CurrencyConverterService/Models/Rates.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Xml.Serialization;
+using Newtonsoft.Json;
+
+namespace CurrencyConverterService.Models
+{
+    class Rates
+    {
+        public int Id { get; set; }
+
+        [JsonProperty(PropertyName = "Cur_ID")]
```

Comment:
Why JsonPropertyAttribute using in DB model?

Reply:
This class is also used to deserialize.